### PR TITLE
LPS-123261 Change log levels: messages should be consistent

### PR DESCRIPTION
**Pull request by robertoDiaz**

### CODE REVIEW

**Comment by robertoDiaz** [comment]
ci:test

**Comment by robertoDiaz** [comment]
ci:test:sf

**Comment by robertoDiaz** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
Just started reviewing :)

File: portal-impl/src/com/liferay/portlet/documentlibrary/messaging/BaseProcessorMessageListener.java [hunk 1]
```diff
@@ -38,6 +38,11 @@ protected void doReceive(Message message) throws Exception {
 		catch (Exception exception) {
 			if (_log.isWarnEnabled()) {
 				_log.warn(
+					"Unable to process file version " +
+						destinationFileVersion.getFileVersionId());
+			}
+			else if (_log.isDebugEnabled()) {
```
Comment (adolfopa) [inline portal-impl/src/com/liferay/portlet/documentlibrary/messaging/BaseProcessorMessageListener.java:44]: Isn't this redundant? If debugEnabled is true, then warnEnabled will be always true. Unless I'm missing something, can you remove the debug logs when there's also an identical warn one?

**Comment by adolfopa** [comment]
@robertoDiaz Can you review my comment? I find the log usage really weird here.

**Pull request closed without merging**